src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -124,101 +99,26 @@ class HaAutomationPicker extends LitElement {
           title: this.hass.localize("ui.card.automation.last_triggered"),
           template: (last_triggered) => html`
             ${last_triggered
-              ? formatDateTime(new Date(last_triggered), this.hass.locale)
+              ? relativeTime(new Date(last_triggered), this.hass.locale)
```

Comment:
We should do relative time if it was <3 days ago, otherwise just short date like "Aug 16"